ansible_playbook.yml

```diff
@@ -0,0 +1,90 @@
+-
+  hosts: all
+  remote_user: vagrant
+  sudo: yes
+  tasks: 
+    - name: "update"
+      apt:
+        update_cache: yes
+    - name: "Install linux-image-extra-"
```

Comment:
I think all those tasks used for installing packages can be replaced with one using `with_items:` clause